pac4j-jwt/pom.xml

```diff
@@ -33,6 +33,10 @@
 			<artifactId>pac4j-core</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>commons-codec</groupId>
```

Comment:
Thinking more about this, we cannot add new methods as the `validate` method is the only one called in the authenticator.
I don't think we need base64 encoding / decoding as JWT tokens are already partially base64 encoded and thus URL safe.
So I would remove the `commons-codec` dependency and fix the test.